brake/backends/cachebe.py

```diff
@@ -71,7 +78,11 @@ def limit(self, func_name, request,
             if ':ip:' in counter:
                 ratelimited_by = 'ip'
 
-            if counters[counter] > count:
+            current_count = counters[counter]
+            if isinstance(current_count, tuple):
```

Comment:
That sounds reasonable to me, but it means that we'll have to wait for a whole version on the old ratelimiting function, which turned out to be pretty bad for our users. Since there's always a window of possible race conditions when upgrading, I'd be in favor of making this a major release and warning people about the possible breakage.

Then again, my upgrades usually take 10 seconds to upgrade every machine, and we're fine with a 10 second window. I don't know if people upgrade things with days between each machine, so I guess it depends on how many people will be impacted.